Access/base_email_access/access.py

```diff
@@ -205,3 +205,6 @@ def fetch_access_approve_email(self, request, data):
 
     def fetch_access_request_form_path(self):
         return "base_email_access/accessRequest.html"
+
+    def email_via_smtp(self, destination, subject, body):
```

Comment:
Can we add a comment on why we have added it like this? `def email_via_smtp` calling helper method of same name - might be confusing in future

Also can we move it from bootprocess.general to `scripts/helpers` or `Access/helpers` - check within team what makes more sense - we will keep all global helper methods (like reading json from file etc) in the same file